src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -64,14 +65,14 @@ object JesBackend {
    * @param wdlValue the value of the input
    * @return a new FQN to WdlValue pair, with WdlFile paths modified if appropriate.
    */
-  private def mapInputGcsPath(wdlValue: WdlValue): WdlValue = {
+  private def mapWdlValueGcsPath(wdlValue: WdlValue): WdlValue = {
```

Comment:
can this be renamed not to begin with "map" so that when it is mapped it doesn't map overload